src/steps/os/unix.rs

```diff
@@ -274,6 +274,11 @@ pub fn run_nix(ctx: &ExecutionContext) -> Result<()> {
         }
     }
 
+    #[cfg(target_os = "macos")]
+    {
+        return Err(SkipStep(String::from("Nix on macOS must be upgraded via darwin-rebuild switch")).into());
```

Comment:
Isn't this effectively disables the nix step in macOS?

Reply:
I realized that this step should be skipped only when nix-darwin is installed. I will fix it later.